libs/redux/src/ducks/globalAlerts.d.ts

```diff
@@ -0,0 +1,43 @@
+import { ADD_GLOBAL_ALERT, ADD_GLOBAL_BANNER, CLOSE_GLOBAL_ALERT, CLOSE_GLOBAL_BANNER } from './globalAlerts'
```

Comment:
this file, the globalAlerts.ts, and change to ducks/index.ts can all be removed once you use the react-hot-toast lib for that one error message alert

Reply:
Alright, I get that updated now. Thanks!